flake.nix

```diff
@@ -64,7 +61,7 @@
           echo 'All metadata checks completed successfully'
         '';
 
-        defaultPackage = pkgs.stdenv.mkDerivation {
+        packages.default = pkgs.stdenv.mkDerivation {
```

Comment:
NB: this is a breaking change for downstream flakes. I think that's fine though.